news/feature-3247.md

```diff
@@ -0,0 +1 @@
+python: added support for arrow syntax
```

Comment:
This is not very talkative, please include where can someone use the arrow syntax, maybe give an example, too.